utils/node_local_test.py

```diff
@@ -1296,6 +1298,68 @@ def log_test(conf,
 
     return lto.fi_location
 
+def set_server_fi(server):
+    """Run the client code to set server params"""
+
+    cmd_env = get_base_env()
+
+    cmd_env['OFI_INTERFACE'] = 'eth0'
+    cmd_env['CRT_PHY_ADDR_STR'] = 'ofi+sockets'
+    vh = ValgrindHelper()
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'conf' in constructor call